excel_import_export/static/src/js/report/action_manager_report.esm.js

```diff
@@ -0,0 +1,55 @@
+/** @odoo-module **/
+
+import {download} from "@web/core/network/download";
+import {registry} from "@web/core/registry";
+
+function getReportUrl({report_name, context, data}, env) {
+    // Rough copy of action_service.js _getReportUrl method.
+    let url = `/report/excel/${report_name}`;
+    const actionContext = context || {};
+    if (data && JSON.stringify(data) !== "{}") {
+        const encodedOptions = encodeURIComponent(JSON.stringify(data));
+        const encodedContext = encodeURIComponent(JSON.stringify(actionContext));
+        return `${url}?options=${encodedOptions}&context=${encodedContext}`;
+    }
+    if (actionContext.active_ids) {
+        url += `/${actionContext.active_ids.join(",")}`;
+    }
+    const userContext = encodeURIComponent(JSON.stringify(env.services.user.context));
+    return `${url}?context=${userContext}`;
+}
+
+async function triggerDownload(action, {onClose}, env) {
+    // Rough copy of action_service.js _triggerDownload method.
+    env.services.ui.block();
+    try {
+        await download({
+            url: "/report/download",
+            data: {
+                data: JSON.stringify([getReportUrl(action, env), "excel"]),
+                context: JSON.stringify(env.services.user.context),
+            },
+        });
+    } finally {
+        env.services.ui.unblock();
+    }
+    if (action.close_on_report_download) {
+        return env.services.action.doAction(
+            {type: "ir.actions.act_window_close"},
+            {onClose}
+        );
+    }
+    if (onClose) {
+        onClose();
+    }
+}
+
+registry
+    .category("ir.actions.report handlers")
+    .add("xlsx_handler", async function (action, options, env) {
```

Comment:
Writing the last comment, found this too.

This is wrong, because it conflicts with another OCA module `report_xlsx`.
It also registers it's handler by the same name here:
https://github.com/OCA/reporting-engine/blob/16.0/report_xlsx/static/src/js/report/action_manager_report.esm.js#L8

If left like this, if you install both modules, only one handler (last installed) will be working and Odoo should be giving errors in the browser DevTools console.

```suggestion
    .add("excel_handler", async function (action, options, env) {
```